test/fixtures/schema/object-fixed.apib

```diff
@@ -6,6 +6,6 @@
 
     + Attributes
         - person (object, fixed)
-            - first_name
-            - last_name
+            - first_name: John
+            - last_name: Smith
```

Comment:
I had deleted the comment previously, but you can disregard this changes since we already have `object-fixed-values.apib` test